### ISSUE TITLE
[Changelog.md] Sort the repositories on page.

### ISSUE THREAD

**Issue by Akhliskun** [mozilla-releng/firefox-infra-changelog#295]
Based on my 1x1 with Jordan, he asked that we sort data based on repo priority.

## Currently our order is like this:
I have added bold to the repos that I consider of priority.
- **autoland**
- ci-admin
- ci-configuration
- **mozilla-beta**
- **mozilla-central**
- **mozilla-inbound**
- **mozilla-release**
- addonscript
- balrogscript
- beetmoverscript
- bouncerscript
- build-cloud-tools
- **build-puppet**
- mozapkpublisher
- **OpenCloudConfig**
- pushapkscript
- pushsnapscript
- scriptworker
- services
- shipitscript
- signingscript
- signtool
- **taskcluster**
- treescript

**Comment by Akhliskun** [comment]
## Proposed order:
- mozilla-release
- mozilla-beta
- mozilla-central
- mozilla-inbound <- Caveat: Will be decommissioned.
- autoland
- build-puppet
- OpenCloudConfig
- Taskcluster
- Script Workers (Alphabetically)
- everything else.

@lundjordan any suggestions on the ordering?

**Comment by Akhliskun** [comment]
From slack:
``` jlund: Can we put all the *script repos together and at the end of the list put in order: autoland, inbound, central, beta, release. ```

**Comment by bccrisan** [comment]
This issue has multiple blockers at the moment.
When analyzing the script I begin asking the following questions:
#### How does the script update the `changelog.md` file?
 - We have `extract_json_from_git()` function that does it
#### Is it worth to modify it?
 - Nope, very complexe..
#### What are the solutions for it?:
 - One solution will be to use the `changelog.json` to generate the md file.

Generating the `changelog.md` file from `changelog.json` would give us control on how we order the above mentioned repositories in `changelog.md` and it would make the code much simpler. At the moment, we call the function whenever we need it, in code, in both git.py and probably hg.py and lists all of the json files in `git_files` and `hg_files`, adds them to a list and loads every json file from the list and adds the content from it to the `changelog.md`. This was done before `changelog.json` was used 

I will create a new issue that should explain what modification we'll need to do in order to generate the `changelog.md` from `changelog.json` and link it here (as a blocker until it's finished).

**Comment by bccrisan** [comment]
Blocked by Issue #302

**Comment by bccrisan** [comment]
So, just to recap, the list is now formed in the following order:
- autoland, 
- mozilla-inbound, 
- mozilla-central, 
- mozilla-beta, 
- mozilla-release.
- build-puppet
- OpenCloudConfig
- addonscript
- pushapkscript
- pushsnapscript
- scriptworker
- shipitscript
- signingscript
- signtool
- treescript
- mozapkpublisher
- services
- build-cloud-tools
- ci-configuration
- ci-admin
